src/vuestic-theme/vuestic-components/va-list/va-list-docs.md

```diff
@@ -0,0 +1,49 @@
+# VaList
+Easy-to-use List component with support for icons, avatars, item selection.
+
+```html
+<va-list fit>
+  <va-list-label color="primary">
+    Customers
+  </va-list-label>
+
+  <va-item clickable to="/foo/1" @click="notify">
+    <va-item-section side>
+      <va-icon name="fa fa-star" />
+    </va-item-section>
+
+    <va-item-section avatar>
+      <va-avatar>
+        <img src="path">
+      </va-avatar>
+    </va-item-section>
+
+    <va-item-section>
+      <va-item-label>
+        Add profile images
+      </va-item-label>
+
+      <va-item-label caption>
+        You can use PNG or JPG files.
+      </va-item-label>
+    </va-item-section>
+  </va-item>
+</va-list>
+```
+
+**VaList Props**
+* `fit` - Boolean - full width
+
+**VaItem Props**
+* `clickable` - Boolean - make list items clickable
+* `to` - String - use item as router-link
+
+**VaListLabel Props**
+* `color`- String (default: info) - theme or color
```

Comment:
I think it's gonna become `primary` instead of `info` if you make changes described above.

Reply:
needs to be info by default

Reply:
Would be nice to consider providing default value to prop itself instead, like `color: {type: String, default: "info"}. However, I'm not sure about possible conflicts with ColorThemeMixin's definition.

Reply:
It'll work just fine `<va-list-label color="warning">`. I think our color mixin takes care of that.